.bamboo/migration-check/execute.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+DB=$1
+
+if [ "$DB" = "mysql" ]; then
+  COMPOSE_FILE="artemis-migration-check-mysql.yml"
+elif [ "$DB" = "postgresql" ]; then
+  COMPOSE_FILE="artemis-migration-check-postgres.yml"
+else
+  echo "Invalid database type. Please choose either mysql or postgresql."
+  exit 1
+fi
+
+# Create libs folder because the Artemis docker compose file expects the .war file there
+mkdir -p build/libs
+mv ./*.war build/libs/
+
+# pass current host's hostname to the docker container for server.url (see docker compose config file)
+export HOST_HOSTNAME=$(hostname)
+
+cd docker
+#just pull everything else than artemis-app as we build it later either way
+docker compose -f $COMPOSE_FILE pull artemis-app $DB
+docker compose -f $COMPOSE_FILE build --build-arg WAR_FILE_STAGE=external_builder --no-cache --pull artemis-app
+docker compose -f $COMPOSE_FILE up --exit-code-from migration-check
+exitCode=$?
+cd ..
+echo "Migration check container exit code: $exitCode"
+if [ $exitCode -eq 0 ]
+then
+    touch .successful
+else
+    echo "Not creating success file because the migration check failed"
+fi
```

Comment:
As this part already led to some discussions, I suggest you remove the creation of the `.successful` file from here and instead add another task for creating this file to both stages in Bamboo.
This also makes it easier in Bamboo to understand where the .successful file comes from.

```suggestion
```

Reply:
Done :)